src/replication.c

```diff
@@ -278,9 +278,9 @@ void replicationFeedMonitors(client *c, list *monitors, int dictid, robj **argv,
     if (c->flags & CLIENT_LUA) {
         cmdrepr = sdscatprintf(cmdrepr,"[%d lua] ",dictid);
     } else if (c->flags & CLIENT_UNIX_SOCKET) {
-        cmdrepr = sdscatprintf(cmdrepr,"[%d unix:%s] ",dictid,server.unixsocket);
+        cmdrepr = sdscatprintf(cmdrepr,"[%d unix:%s%s%s] ",dictid,server.unixsocket, c->name ? " " : "", c->name ? (char*)c->name->ptr : "");
```

Comment:
Maybe limit the size of the client name. Also I think it's ok to print an extra space if there's no client name and it makes the code simpler.
```suggestion
        cmdrepr = sdscatprintf(cmdrepr,"[%d unix:%s %.20s] ",dictid,server.unixsocket, c->name ? (char*)c->name->ptr : "");
```

Reply:
I think we moved this entire conversation to a new issue: https://github.com/redis/redis/pull/6832 since we talked about having a new monitor format.